chameleonultragui/lib/gui/page/debug.dart

```diff
@@ -58,6 +59,98 @@ class DebugPage extends StatelessWidget {
               Text(
                   '${localizations.chameleon_device_type}: ${appState.connector!.device}'),
+              Text(
+                  '${localizations.shared_preferences_logging}: ${appState.sharedPreferencesProvider.isDebugLogging()} with ${appState.sharedPreferencesProvider.getLogLines().length} lines'),
+              const SizedBox(height: 10),
+              Text(localizations.production_logging, textScaleFactor: 1.5),
+              const SizedBox(height: 5),
+              Text(localizations.slow_down_warning, textScaleFactor: 0.9),
+              const SizedBox(height: 10),
+              if (!appState.sharedPreferencesProvider.isDebugLogging())
+                ElevatedButton(
+                  onPressed: () async {
+                    appState.sharedPreferencesProvider.setDebugLogging(true);
+                    await appState.connector!.performDisconnect();
+                    appState.log = null;
+                    appState.connector = null;
+                    appState.changesMade();
+                  },
+                  child: Column(children: [
+                    Text(localizations.enable_production_logging),
+                  ]),
+                ),
+              if (appState.sharedPreferencesProvider.isDebugLogging())
+                ElevatedButton(
+                  onPressed: () async {
+                    appState.sharedPreferencesProvider.setDebugLogging(false);
+                    await appState.connector!.performDisconnect();
+                    appState.log = null;
+                    appState.connector = null;
+                    appState.changesMade();
+                  },
+                  child: Column(children: [
+                    Text(localizations.disable_production_logging),
+                  ]),
+                ),
+              const SizedBox(height: 10),
+              ElevatedButton(
+                onPressed: () async {
+                  appState.sharedPreferencesProvider.clearLogLines();
+                  appState.changesMade();
+                },
+                child: Column(children: [
+                  Text(localizations.clear_logs),
+                ]),
+              ),
+              const SizedBox(height: 10),
+              ElevatedButton(
+                onPressed: () async {
+                  await Clipboard.setData(ClipboardData(
+                      text: appState.sharedPreferencesProvider
+                          .getLogLines()
+                          .join("\n")));
+                },
+                child: Column(children: [
+                  Text(localizations.copy_logs_to_clipboard),
+                ]),
+              ),
+              const SizedBox(height: 10),
+              ElevatedButton(
+                onPressed: () async {
+                  var resp = json.decode((await http.post(
+                    Uri.parse('https://hastebin.skyra.pw/documents'),
```

Comment:
this app aint gonna post to anywhere. They should just use copy to clipboard and paste it into their pastebin of choice or just directly upload it to gh as a file.

Reply:
Remember your cors proxy?

Reply:
1) it doesnt exist / isnt being used
2) A lot of websites got sued because they used fing google fonts and google keeps a log of the IP aress, nothing more. I dont want to be sued because the app uploads fcking entire logs possibly containing senstive info on some random 3rdparty service